app/schemas/models/user.coffee

```diff
@@ -346,6 +346,16 @@ _.extend UserSchema.properties,
   role: {type: 'string', enum: ["God", "advisor", "parent", "principal", "student", "superintendent", "teacher", "technology coordinator"]}
   birthday: c.stringDate({title: "Birthday"})
   lastAchievementChecked: c.stringDate({ name: 'Last Achievement Checked' })
+  signupQueryParams: { type: 'object', properties: {
```

Comment:
Just for Israel, but figured it could be used for others as well, so didn't want to restrict it by name. A description would be good, though.